dockerfiles/verify_packages/installer/utils.sh

```diff
@@ -48,3 +58,7 @@ install_legacy_ddtrace() {
     tar -xf  "/tmp/legacy-${version}.tar.gz" -C /
     /opt/datadog-php/bin/post-install.sh
 }
+
+get_php_conf_dir() {
+    php -i | grep -i 'scan this dir for additional .ini files' | awk '{print $NF}'
```

Comment:
I consider it my duty to educate people on AWK, so this is equivalent:

```
php -i | awk 'tolower($0) ~ /scan this dir for additional .ini files/ {print $NF}'
```

You don't have to change it; just an FYI ^_^

I don't think case insensitive matching is really necessary, so this would also suffice:

```
php -i | awk '/Scan this dir for additional .ini files/ {print $NF}'
```

Reply:
I appreciate this `awk` "advanced" usage (advanced if compared to my knowledge of it 😄 ). If you don't mind I'd prefer to stick with the existing implementation, that even if uses two tools instead of one I think is more immediate the the average reader (and by average I mostly mean myself 😃 )